extension/inet/include/html_charref.hpp

```diff
@@ -0,0 +1,4552 @@
+		mapped_strings[string_t(owned_strings[288])] = 120079;
+		mapped_strings[string_t(owned_strings[289])] = 8920;
+		mapped_strings[string_t(owned_strings[290])] = 866;
+		mapped_strings[string_t(owned_strings[291])] = 319;
+		mapped_strings[string_t(owned_strings[292])] = 10229;
+		mapped_strings[string_t(owned_strings[293])] = 1023;
+		mapped_strings[string_t(owned_strings[294])] = 10230;
+		mapped_strings[string_t(owned_strings[295])] = 10232;
+		mapped_strings[string_t(owned_strings[296])] = 1023;
+		mapped_strings[string_t(owned_strings[297])] = 10233;
+		mapped_strings[string_t(owned_strings[298])] = 120131;
+		mapped_strings[string_t(owned_strings[299])] = 860;
+		mapped_strings[string_t(owned_strings[300])] = 8600;
+		mapped_strings[string_t(owned_strings[301])] = 8466;
+		mapped_strings[string_t(owned_strings[302])] = 862;
+		mapped_strings[string_t(owned_strings[303])] = 321;
+		mapped_strings[string_t(owned_strings[304])] = 8810;
+		mapped_strings[string_t(owned_strings[305])] = 1050;
+		mapped_strings[string_t(owned_strings[306])] = 1052;
+		mapped_strings[string_t(owned_strings[307])] = 8287;
+		mapped_strings[string_t(owned_strings[308])] = 849;
+		mapped_strings[string_t(owned_strings[309])] = 120080;
+		mapped_strings[string_t(owned_strings[310])] = 8723;
+		mapped_strings[string_t(owned_strings[311])] = 12013;
+		mapped_strings[string_t(owned_strings[312])] = 8499;
+		mapped_strings[string_t(owned_strings[313])] = 924;
+		mapped_strings[string_t(owned_strings[314])] = 103;
+		mapped_strings[string_t(owned_strings[315])] = 323;
+		mapped_strings[string_t(owned_strings[316])] = 327;
+		mapped_strings[string_t(owned_strings[317])] = 32;
+		mapped_strings[string_t(owned_strings[318])] = 1053;
+		mapped_strings[string_t(owned_strings[319])] = 8203;
+		mapped_strings[string_t(owned_strings[320])] = 820;
+		mapped_strings[string_t(owned_strings[321])] = 8203;
+		mapped_strings[string_t(owned_strings[322])] = 8203;
+		mapped_strings[string_t(owned_strings[323])] = 881;
+		mapped_strings[string_t(owned_strings[324])] = 8810;
+		mapped_strings[string_t(owned_strings[325])] = 10;
+		mapped_strings[string_t(owned_strings[326])] = 12008;
+		mapped_strings[string_t(owned_strings[327])] = 8288;
+		mapped_strings[string_t(owned_strings[328])] = 160;
+		mapped_strings[string_t(owned_strings[329])] = 846;
+		mapped_strings[string_t(owned_strings[330])] = 10988;
+		mapped_strings[string_t(owned_strings[331])] = 8802;
+		mapped_strings[string_t(owned_strings[332])] = 881;
+		mapped_strings[string_t(owned_strings[333])] = 8742;
+		mapped_strings[string_t(owned_strings[334])] = 8713;
+		mapped_strings[string_t(owned_strings[335])] = 880;
+		mapped_strings[string_t(owned_strings[336])] = 8770;
+		mapped_strings[string_t(owned_strings[337])] = 8708;
+		mapped_strings[string_t(owned_strings[338])] = 881;
+		mapped_strings[string_t(owned_strings[339])] = 8817;
+		mapped_strings[string_t(owned_strings[340])] = 8807;
```

Comment:
I still think this isn't correct, if we look at the spec:
<https://html.spec.whatwg.org/multipage/named-characters.html>
It lists `&NotGreaterFullEqual` as `U+02267 U+00338`, with a glyph that matches the output of python's `html.escape` method:
```
>>> import html
>>> html.unescape('&NotGreaterFullEqual;')
'≧̸'
```

As said before, we should make a list of the entries that create 2(or more if that's possible?) codepoints and throw a NotImplementedException when we encounter them for now

Reply:
Let's add proper support for multi-codepoint escapes? There appear to be quite a few of them.

On the plus side - there at most two codepoints, so we can represent this as a fixed-size struct, e.g.:

```sql
struct HTMLEscapeCodepoint {
    uint32_t primary_codepoint;
    uint32_t secondary_codepoint; // 0 if there is only one codepoint
};
```